apps/expo-go/android/expoview/src/main/java/host/exp/exponent/experience/splashscreen/ManagedAppSplashScreenConfiguration.kt

```diff
@@ -66,6 +66,16 @@ class ManagedAppSplashScreenConfiguration private constructor() {
      * - generic splash imageUrl
      */
     private fun parseImageUrl(manifest: Manifest): String? {
+      val pluginInfo = manifest.getPluginProperties("expo-splash-screen")
+      pluginInfo?.let { info ->
+        val url = manifest.getIconUrl()?.split("./")?.get(0)
```

Comment:
let's use the icon url for the loading view in expo go and expo-dev-client instead of the splash config

Reply:
will need to consider how this works for adaptive icon / ios light/dark/tinted (probably just use light for now). we can follow up later with a loadingView key in app.json for customizing this behavior.